src/components/common/CategoriesCards/CategoryCard.jsx

```diff
@@ -14,12 +18,13 @@ const CategoryCard = ({
     title === 'Without Category'
```

Comment:
Still dependency on text content. Not good. You can pass `key` to the category card and use it for condition